src/sql/strong-oracle/create.sql

```diff
@@ -0,0 +1,332 @@
+BEGIN
```

Comment:
I'm not qualified to review this; I think Abraham and/or Nara should take a look. A quick overview of the `CREATE INDEX` and `ALTER TABLE` statements would be helpful though. For example, are `XPKL_SUBSCRIPTIONS` and `PK_ACCOUNTS` oracle naming conventions?